res/controllers/American Audio VMS4.midi.xml

```diff
@@ -268,27 +268,39 @@
             <status>0x80</status>
             <midino>0x0C</midino>
          </control>
+         <control><!-- on -->
+            <group>[Channel1]</group>
+            <key>cue_play</key>
+            <status>0x91</status>
+            <midino>0x0C</midino>
+         </control>
+         <control><!-- off -->
+            <group>[Channel1]</group>
+            <key>cue_play</key>
+            <status>0x81</status>
+            <midino>0x0C</midino>
+         </control>
          <control><!-- on -->
             <group>[Channel1]</group>
             <key>play</key>
             <status>0x90</status>
             <midino>0x0D</midino>
          </control>
          <control><!-- off -->
-             <group>[Channel1]</group>
-             <key>play</key>
-             <status>0x80</status>
-             <midino>0x0D</midino>
+            <group>[Channel1]</group>
+            <key>play</key>
+            <status>0x80</status>
+            <midino>0x0D</midino>
          </control>
          <control><!-- Shifted Play -->
             <group>[Channel1]</group>
-            <key>start_play</key>
+            <key>sync_master</key>
```

Comment:
Why is this mapped? @ywwg why does this Control exist? Should we remove it to avoid future confusion?

Reply:
I mapped as many things from the Traktor Edition ov the VMS4 to the standard one that did not conflict with how the standard one is labeled. This is one of those items.

Reply:
It's on the MixxxControls wiki page. @ywwg what say you?